### DISCUSSION TITLE
Make Auto Input/Output be checked by default

### DISCUSSION THREAD

**Discussion by Ayutac** [TechReborn/TechReborn#2902] [Ideas]
The config should contain an option to make Auto Input/Output of machines and storages checked by default because some players would prefer that

**Comment by Qrsmith** [comment]
Thanks for pointing me here. 

Seems to me that it should be relatively simple to swap these to checked by default. 
There are many many more situations where you would want machines automatically inputting/outputting than not, two extra clickers per machine isn't a big deal either way- but it definitely adds up.

**Comment by aria1th** [comment]
Slot Configurations are laggy enough, especially with auto input / output... rather, it might be better to just provide sided inventories passively at default, just as tanks do. 
Or better Copy-Paste, such as wrench functionality might be still better to configure repeated machines.

**Reply by Ayutac** [reply]
that reminds me, didn't @Technici4n say something about how the inventory system in TR is basically bad the way it is right now?

**Reply by Technici4n** [reply]
Well it has one important feature: it works

**Reply by aria1th** [reply]
most of the problem comes from active machines, machines will work and try to balance recipe, search recipes even if they are not powered,  and especially auto crafting table /electric furnace does not have 'fixed' recipe option to cache. I'm pretty sure it can be 70% better than now, by having option to fix recipes in block entity itself, not searching recipe every tick...

but it works anyway.
